### ISSUE TITLE
OF not building for arm64e on Gen 3 iPad Pro

### ISSUE THREAD

**Issue by sethismyfriend** [openframeworks/openFrameworks#6198]
I have one of the 3rd Generation iPad Pros which has the arm64e architecture. When building OF during the linking phase of compliation many of the internal libraries are not compiling. I tried reverting the architecture to arm64 but then was not able to deploy to the application because the architecture does not match the A12 chip on the iPad Pro. 

I know these devices were just released a month or so ago - and I'm wondering if all these libraries have to be recompiled in order for OF to work on the latest iPad?  

Showing Recent Messages
ld: warning: ignoring file ../../../libs/boost/lib/ios/boost_filesystem.a, missing required architecture arm64e in file ../../../libs/boost/lib/ios/boost_filesystem.a (4 slices)ld: warning: ignoring file ../../../libs/FreeImage/lib/ios/freeimage.a, missing required architecture arm64e in file ../../../libs/FreeImage/lib/ios/freeimage.a (4 slices)ld: warning: ld: warning: ignoring file ../../../libs/tess2/lib/ios/tess2.a, missing required architecture arm64e in file ../../../libs/tess2/lib/ios/tess2.a (4 slices)

**Comment by arturoc** [comment]
yes we need to add arm64 target to the each library to support this.

**Comment by danoli3** [comment]
I'll jump on this on Apothecary @arturo

**Comment by arturoc** [comment]
@sethismyfriend yes we need to compile every library for the new target, it's not trivial, it's usually a headache with some of the libraries. we have scripts for building the libraries in the apothecary repository if you want to take a look but even with that adding a new platform or even architecture like in this case is problematic

@danoli3 thanks!!

**Comment by sethismyfriend** [comment]
Thanks for the clarification - I figured it would be non-trivial.

**Comment by sethismyfriend** [comment]
I can help test.

**Comment by danoli3** [comment]
Awesome yeah I hope to get to this today :)!
